docs/source/api/solution.rst

```diff
@@ -1,20 +1,21 @@
 .. _ref_api_solution:
 
 ***************
-Solution object
+``DpfSolution``
```

Comment:
```suggestion
``DpfSolution`` object
```

Reply:
Same here. Objects are "instances" of a class. An object should not have an individual definition, only the class.